artifacts/deployment/README.md

```diff
@@ -1,26 +1,50 @@
-# `EtcdProxyController` manifest
-
-This directory contains manifest for deploying the `EtcdProxyController` in the cluster. It creates the following resources:
-* **namespace/kube-apiserver-storage** - namespace where controller and resources managed by controller are deployed.
-* **serviceaccount/etcdproxy-controller-sa** - ServiceAccount used by Controller for managing EtcdStorage objects, ReplicaSets and Services.
-* **serviceaccount/etcdproxy-sa** - ServiceAccount used by EtcdProxy Pods.
-* **clusterrole/etcdproxy-crd-clusterrole** - ClusterRole allowing **etcdproxy-controller-sa** ServiceAccount to update EtcdStorage status.
-* **role/etcdproxy-controller-role** - Role allowing **etcdproxy-controller-sa** ServiceAccount to manage ReplicaSets, Services, ConfigMap and Secrets in the **kube-apiserver-storage** namespace.
+# Deploying `EtcdProxy` Controller
+
+This directory contains manifest for deploying the `EtcdProxy` Controller in the cluster. The manifest deploys the following resources:
+
+* **namespace/kube-apiserver-storage** - namespace for the EtcdProxyController to lives in,
+
+* **sa/etcdproxy-controller-sa** - ServiceAccount for managing EtcdStorage objects, ReplicaSets and Services, Secrets and ConfigMaps,
+* **sa/etcdproxy-sa** - ServiceAccount used by EtcdProxy Pods. Does not have any permissions,
+
+* **clusterrole/etcdproxy-crd-clusterrole** - ClusterRole for managing EtcdStorages.
 * **clusterrolebinding/etcdproxy-crd-clusterrolebinding** - Binds **clusterrole/etcdproxy-crd-clusterrole** to **serviceaccount/etcdproxy-controller-sa**.
+
+* **role/etcdproxy-controller-role** - Role for managing ReplicaSets, Services, ConfigMap and Secrets in the **kube-apiserver-storage** namespace.
 * **rolebinding/etcdproxy-controller-rolebinding** - Binds **role/etcdproxy-controller-role** to **serviceaccount/etcdproxy-controller-sa**.
-* **customresourcedefinition/etcdstorages.etcd.xmudrii.com** - EtcdStorage CRD used to manage etcd proxies.
-* **deployment/etcdproxy-controller-deployment** - Deployment for the `EtcdProxyController`.
 
-## RBAC Rules
+* **customresourcedefinition/etcdstorages.etcd.xmudrii.com** - CRD defining the EtcdStorage type for managing etcd proxies,
+* **deployment/etcdproxy-controller-deployment** - Controller Deployment.
+
+## Deploying the core `etcd`
+
+The deployment manifest assumes you have the core `etcd` deployed and exposed on `https://etcd-svc-1.etcd.svc:2379`.
+The URL can be changed by modifying the `--etcd-core-url` flag.
+
+The core `etcd` can be deployed using the manifests located in the [`artifacts/etcd`](https://github.com/xmudrii/etcdproxy-controller/tree/master/artifacts/etcd) directory.
```

Comment:
There is an example etcd deployment located ...

Reply:
Done.